backend/app/models/enrollment.rb

```diff
@@ -122,13 +122,25 @@ def mark_event_as_read(event_name)
   end
 
   def notify_events_from_demandeurs_count
+    events.where(
+      name: "notify",
+      user_id: demandeurs.pluck(:user_id)
+    ).count
+  end
+
+  def unprocessed_notify_events_from_demandeurs_count
     events.where(
       name: "notify",
       processed_at: nil,
       user_id: demandeurs.pluck(:user_id)
     ).count
   end
 
+  def recent
```

Comment:
Hum `recent_unprocessed_submit_event` is more detailed but also misleading as it may suppose having multiple submit events. But I agree that we can find something better than just `recent`.